package.nix

```diff
@@ -231,6 +230,8 @@ in {
   ] ++ lib.optionals buildUnitTests [
     gtest
     rapidcheck
+  ] ++ lib.optionals enableManual [
+    man
```

Comment:
Okay, done.

My rational was, that man is needed for displaying the man-page anyway so i did put it into BuildInputs as i thought it will be a runtime dependency. Or am i wrong about this? I mean this code: https://github.com/NixOS/nix/blob/79bbb4a64cb2a1cde48da93718c64352cab61046/src/libmain/shared.cc#L319
Or did i miss something?

Reply:
Adding it buildInputs doesn't make it a runtime dependency automatically. Using something like `makeWrapper` does.

@roberth also why `nativeBuildInputs` and not `checkInputs`?

Reply:
Thanks for the clarification!
The help command will fail if man is disabled in the config then. 
Should I add a wrapper or just leave it as it is and make this an issue for the future?

Reply:
No I wouldn't add a wrapper and keep it an optional run time dependency for now.

Reply:
> why `nativeBuildInputs` and not `checkInputs`?

Because I haven't internalized this 2 year old parameter yet, but then it should be `nativeInstallCheckInputs`. That's the most accurate role for this, and iiuc it would let us get rid of the `lib.optionals doInstallCheck [` bit in `nativeBuildInputs`.

Let's fix it in the old way first, and then we can refactor it more later without increasing the scope of this PR.